rllib/core/learner/torch/torch_learner.py

```diff
@@ -241,13 +247,23 @@ def build(self) -> None:
         # register them in the MultiAgentRLModule. We should find a better way to
         # handle this.
         if self._distributed:
+            # TODO (sven): Shouldn't self._module always be MARL?
             if isinstance(self._module, TorchRLModule):
                 self._module = TorchDDPRLModule(self._module)
             else:
+                assert isinstance(self._module, MultiAgentRLModule)
                 for key in self._module.keys():
-                    if isinstance(self._module[key], TorchRLModule):
+                    sub_module = self._module[key]
+                    if isinstance(sub_module, TorchRLModule):
+                        ddp_class = (
+                            TorchDDPRLModuleWithTargetNetworksInterface
+                            if isinstance(
```

Comment:
can we instead separate out this wrapping function and wrap with `TorchDDPRLModuleWithTargetNetworksInterface` in the appo learner? I feel like this doesn't belong here.

Reply:
we could add an extra private method on torch learner and place it there. That would be much cleaner